ijroi/ijroi.py

```diff
@@ -89,17 +89,31 @@ def getfloat():
     stroke_color = get32()
     fill_color = get32()
     subtype = get16()
-    if subtype != 0:
-        raise NotImplementedError('roireader: ROI subtype %s not supported (!= 0)' % subtype)
+    
     options = get16()
     arrow_style = get8()
     arrow_head_size = get8()
     rect_arc_size = get16()
     position = get32()
     header2offset = get32()
 
+    # End Header data
+    #===========================================================================
+
+    #RoiDecoder.java#L177
+    subPixelResolution = ((options&SUB_PIXEL_RESOLUTION)!=0) and (version>=222)
```

Comment:
I see this in the ImageJ source, but can you explain why the version check is necessary? Which version is 222? Is it older than 2012? (This code was, and it should have worked then.)

Let's not make any references to line numbers in ImageJ master because they'll change.

Reply:
I actually don't know, I saw it in the java file and figured if it's a feature of the ImageJ decoder it should be a feature of this decoder too. Happy to remove that change if you'd like.

Reply:
Looks like that dates back to the original implementation in 2011, so that's fine: https://github.com/imagej/imagej1/blame/f8e5dfd74313d4e05dae8c6ae35965dc5ce02981/ij/io/RoiDecoder.java#L177